components/contribute-cards/Contribute.js

```diff
@@ -212,9 +228,18 @@ const ContributeCard = ({
         <Box>
           {!disableCTA && (
             <Link href={route}>
-              <StyledButton buttonStyle={getCTAButtonStyle(type)} width={1} mb={2} mt={3} data-cy="contribute-btn">
+              <CTAButton
```

Comment:
The issue was just about the card hero, I don't think we want to customize the button

Reply:
Reverted the styles on button 👍🏻